aidial_interceptors_sdk/chat_completion/adapter.py

```diff
@@ -49,6 +51,15 @@ async def chat_completion(
             )
 
             request_body = await request.original_request.json()
+
```

Comment:
1. Extract this into a dedicated method + add a comment that `openai` requires `model` parameter even for `AsyncAzureOpenAI`
2. Use the method for the embeddings as well